app/models/company.rb

```diff
@@ -58,6 +58,8 @@ def last_char_star?
   end
 
   def refresh_search_suggestions
+    # It feels weird that you're calling a class method directly on a concern.
+    # Could this be a service object?
```

Comment:
Ya, I've been meaning to move the entire `SearchSuggestion` module out of `app/models/concerns` since it has its own routing endpoint & controller. Though, I'd prefer to keep API in the class scope. I'm not sure what you mean by 'service object'.